customerRegistration/src/main/java/DAO/DBConnection.java

```diff
@@ -0,0 +1,41 @@
+package DAO;
+
+import java.io.FileInputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DBConnection {
+
+	private String jdbcUrl = "jdbc:mysql://localhost:3306/hotwaxsystem?useSSL=false";
+	private String jdbcDriver="com.mysql.cj.jdbc.Driver";
+
+	protected Connection getConnection() throws Exception {
+
+		//Connection connection = null;
+
+		//try {
+			Properties propCred = new Properties();
+    		propCred.load(new FileInputStream("/home/ayan/eclipse-workspace/customerRegistration/data.properties"));
```

Comment:
This is bad practice. What if your code is running on a different machine? On that machine /home/ayan folder/user will not be present. You should find the root location of the project by the use of supporting methods. And then from the project root location you will navigate to the project specific internal directories.

Reply:
Corrected